### PR TITLE
Feature/updated feed service

### PR DESCRIPTION
**Pull request by dev-swapnildixit4161**

### CODE REVIEW

File: feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java [hunk 1]
```diff
@@ -3,23 +3,22 @@
 import com.nashtech.feedservice.model.Nasher;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.apache.poi.ss.usermodel.Cell;
-import org.apache.poi.ss.usermodel.Row;
-import org.apache.poi.ss.usermodel.Sheet;
```
Comment (abidknashtech) [inline feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java:8]: don't (*) use single import

File: feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java [hunk 2]
```diff
@@ -3,23 +3,22 @@
...
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 import java.util.*;
 
 public class ExcelHelper {
     private static final Logger logger = LogManager.getLogger(ExcelHelper.class);
     private static final String TYPE = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
```
Comment (abidknashtech) [inline feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java:20]: please revert the Header as previously used.

**Review by abidknashtech — APPROVED** [review]

File: feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java (hunk 1 above)
Comment (dev-swapnildixit4161) [reply feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java:8]: Done

File: feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java (hunk 2 above)
Comment (dev-swapnildixit4161) [reply feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java:26]: Done

**Pull request closed without merging**